app/models/maintenance_stats/queries/github/repository_contributor_stats_query.rb

```diff
@@ -5,22 +5,20 @@ module Github
       class RepositoryContributorStatsQuery < BaseQuery
         VALID_PARAMS = [:full_name]
         REQUIRED_PARAMS = [:full_name]
-        RETRY_AMOUNT = 4
+        TIMEOUT_SEC = 10
 
         def self.client_type
           :v3
         end
 
-        def query(params: {}, count: 0)
-          raise Octokit::Error.new "Didn't get a response after #{RETRY_AMOUNT} attempts" if params[:count].present? && params[:count] >= RETRY_AMOUNT
+        def query(params: {})
           validate_params(params)
 
-          resp = @client.contributor_stats(params[:full_name])
-          if @client.last_response.status == 202
-            sleep (count + 1) * 0.5
-            return query(params: params, count: count+=1)
-          end
-          resp
+          result = @client.contributor_stats(params[:full_name], retry_timeout: TIMEOUT_SEC)
```

Comment:
Good find on the built-in parameter!

Reply:
If only I didn't spend a bunch of time fixing the existing check first 😆